src/stores/Frame/FrameStore.ts

```diff
@@ -333,6 +336,19 @@ export class FrameStore {
         }
     }
 
+    @computed get requiredUnit() {
```

Comment:
Would it be more clear if we rename `frame.unit` to `frame.headerUnit` to show the difference between unit and requiredUnit? In addition shall we remove `@computed` for frame.unit? There seems no observables in the function.

Reply:
great suggestion. thanks